content/en/blog/_posts/2018-12-11-Kubernetes-Federation-Evolution.md

```diff
@@ -82,7 +82,7 @@ For external clients, automatic DNS expansion described is not currently possibl
 
 That way, your clients can always use the short form on the left, and always be automatically routed to the closest healthy shard on their home continent. All of the required failover is handled for you automatically by Kubernetes cluster federation.
 
-As further reading, a more elaborate example for users is available in the [Multi-Cluster Service DNS with ExternalDNS guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/servicedns-with-externaldns.md).
+As further reading, a more elaborate example for users is available in the [Multi-Cluster Service DNS with ExternalDNS guide](https://github.com/kissingwolf/federation-v2/blob/master/docs/ingressdns-with-externaldns.md#multi-cluster-ingress-dns-with-externaldns-guide).
```

Comment:
Since the link no longer exists, and I don't believe there is and equivalent link,  my recommendation would be remove the whole sentence referencing the link.  Also assigning a blog tech lead to confirm.
/assign @onlydole

Reply:
Yes the link no longer exists other than the old forked link as reviewed above.  so I also recommend to remove the whole sentence referencing the link.